inference/core/interfaces/http/http_api.py

```diff
@@ -1200,6 +1206,45 @@ async def model_add(dataset_id: str, version_id: str, api_key: str = None):
                     }
                 )
 
+        if not LAMBDA:
+
+            @app.get(
```

Comment:
probably should be post?

Reply:
not sure how that would work with redirect

Reply:
Ya, I'm not sure how the redirect works with POST. I'm going to leave it as GET for now since it's working and we don't expect users to be invoking this route other than through our own landing page.